### ISSUE TITLE
manual mesh definition

### ISSUE THREAD

**Issue by jackshen2018** [usnistgov/fipy#501]
Does fipy support manual mesh definition from numpy arrays? I did not find any information about this in the user manual, can anybody help me with this?

**Comment by wd15** [comment]
Yes, it is possible to specify meshes from Numpy arrays. The following mailing list question may help you. It uses Scipy's Dealunay triangulation to create a mesh rather than Gmsh, but I think it will help you figure out how to specify the vertices, faces and cells to FiPy's mesh object

 https://www.mail-archive.com/fipy@nist.gov/msg03691.html
